components/atom/button/src/index.scss

```diff
@@ -188,5 +188,9 @@ $p-atom-button-large: $p-l !default;
 
   &--link {
     text-decoration: none;
+
+    &:hover {
```

Comment:
Without duplicating code:
```
  &--link {
    &,
    &:hover {
      text-decoration: none;
    }
  }
```

Reply:
I did that at the beggining but I though it was a bit confusing. :D